src/Store/CookieStore.php

```diff
@@ -26,231 +26,257 @@ class CookieStore implements StoreInterface
 
     /**
      * Cookie base name.
-     * Use 'cookiePrefix' argument to set this during instantiation.
+     * Use 'namespace' argument to set this during instantiation.
      */
-    private string $cookiePrefix;
+    private string $namespace;
 
     /**
-     * Number of bytes to deduct/buffer from KEY_CHUNKING_THRESHOLD before chunking begins.
+     * The threshold (in bytes) in which chunking/splitting occurs.
      */
-    private int $chunkingThreshold;
+    private int $threshold;
+
+    /**
+     * Internal cache of the storage state.
+     *
+     * @var array<mixed>
+     */
+    private array $store = [];
+
+    /**
+     * When true, CookieStore will not setState() itself. You will need manually call the method to persist state to storage.
+     */
+    private bool $deferring = false;
 
     /**
      * CookieStore constructor.
      *
      * @param SdkConfiguration $configuration   Base configuration options for the SDK. See the SdkConfiguration class constructor for options.
-     * @param string           $cookiePrefix    A string to prefix stored cookie keys with.
+     * @param string           $namespace       A string in which to store cookies under on devices.
      */
     public function __construct(
         SdkConfiguration $configuration,
-        string $cookiePrefix = 'auth0'
+        string $namespace = 'auth0'
     ) {
-        [$cookiePrefix] = Toolkit::filter([$cookiePrefix])->string()->trim();
+        [$namespace] = Toolkit::filter([$namespace])->string()->trim();
 
         Toolkit::assert([
-            [$cookiePrefix, \Auth0\SDK\Exception\ArgumentException::missing('cookiePrefix')],
+            [$namespace, \Auth0\SDK\Exception\ArgumentException::missing('namespace')],
         ])->isString();
 
         $this->configuration = $configuration;
-        $this->cookiePrefix = $cookiePrefix ?? 'auth0';
+        $this->namespace = hash(self::KEY_HASHING_ALGO, $namespace ?? 'auth0');
```

Comment:
No need for `?? 'auth0` here. It will `$namespace` will never be null, right?

https://3v4l.org/n2sMC

Reply:
You are correct that this will never really be `null`; this was a workaround for a static code analysis warning (since the `filter()->string()-trim()` method can technically return null; it'sa nullsafe trim that ignores when the value is null) that I should have used an ignore flag for instead. Fixed!